nfc/libs/NfcCoreLib/lib/NciCore/phNciNfc_DbgDescription.c

```diff
@@ -306,13 +308,15 @@ void phNciNfc_PrintNfceeModeSetCmdDescription(uint8_t *pBuff, uint16_t wLen)
 
 void phNciNfc_PrintCoreResetRspDescription(uint8_t *pBuff, uint16_t wLen)
 {
-    PHNCINFC_VALIDATE_PACKET_LENGTH(3, wLen);
+    PHNCINFC_VALIDATE_PACKET_LENGTH(1, wLen);
     PH_LOG_NCI_INFO_STR("Status: %!NCI_STATUS!", pBuff[0]);
+    PHNCINFC_VALIDATE_PACKET_LENGTH(3, wLen);
```

Comment:
There are 2 checks because the NCI_CORE_RESET_RSP may have 2 different formats depending of
the current NCI version running on the NFC Controller.
- In NCI 1.x the format is 1 byte for status / 1 byte for NCI version / 1 byte for Configuration Status
- In NCI 2.0 the format is 1 byte for status.

I will had a comment to make this clearer. I will also use macro instead of 1 and 3.